modules/sw360/src/main/java/org/eclipse/sw360/antenna/workflow/generators/SW360Updater.java

```diff
@@ -45,16 +45,24 @@ public SW360Updater() {
     @Override
     public void configure(Map<String, String> configMap) throws AntennaConfigurationException {
         SW360ProjectCoordinates configuredSW360Project = context.getConfiguration().getConfiguredSW360Project();
-        projectName = configuredSW360Project.getName();
```

Comment:
I am not a fan of this solution, it is a meaningless spagetti code solution. How about something like
Optional\<Tinaconfig\> configuration  = getTinaconfig();
projectName = retrieveName(configuration);
projectVersion = retrieveVersion(configuration);

String retrieveName(Optional\<Tinaconfig\> configuration) {
    if configuration.isPresent()
        return configuration.getName();
    else
        return context...